feature/bookmarks/src/main/res/drawable/img_empty_bookmarks.xml

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Mentioned in chat, let's keep this as an SVG if possible for better resizing support.